src/app/page.module.css

```
@@ -27,35 +35,264 @@
 
-.heroSection > h1{
-    font-size: 20px;
 }
 
+.heroSection p{
+    font-size: 1.75rem;
+    font-weight: 500;
+    font-style: medium;
+    line-height: 1.4;
+    color: var(--text-secondary);
+    font-family: 'Poppins', sans-serriff;
```

Biome (2.1.2)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)